src/js/auth/register.js

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
This looks exactly like the login functionality. Why can't you just stick to one and remove the other? You can rename the loginUser function to a common name such as `fetch_auth`